src/app/components/Home/ListOfAddresses/components/index.ts

```diff
@@ -0,0 +1,5 @@
+import LinkOfAdress from "./LinkOfAdress";
+import ListItems from "./ListItems";
+import NavListOfAddress from "./NavListOfAddress";
+
```

Comment:
Or you can simply `export { default as LinkOfAddress } from "./LinkOfAddress"` etc.